### ISSUE TITLE
.eps export for figures

### ISSUE THREAD

**Issue by Bonnevie** [ioam/holoviews#2787]
I know that matplotlib allows exporting to .eps files, which is used a lot by people working with LaTeX since it is one of the few vector graphics formats that are reasonably well supported via standard packages.
Since I assume you are using matplotlib's interface to export to png/svg in the background, would it be difficult to add the .eps option?  
Is there a workaround for accessing the generated matplotlib figure and save it directly using the backend?
Of course, it would be great if this also worked for the other backends, but I am unsure about their support.

**Comment by jlstevens** [comment]
I'll have to check whether eps is supported for saving out files by the matplotlib renderer (it should be). The general issue with .eps for rendering for interactive work is that web browsers don't render eps properly afaik which is why we tend to use PNG for raster output and SVG for vector output. We tried PDF output at once point but that didn't integrate well with web browsers either so we dropped that iirc.

So in short, I'll check if EPS can be output to file using the explicit renderer API but I don't see .eps being a valid output in the browser for interactive work.

**Comment by philippjfr** [comment]
>Is there a workaround for accessing the generated matplotlib figure and save it directly using the backend?

Yes there is using, e.g.:

```python
obj = hv.Curve(range(10))
fig = hv.renderer('bokeh').get_plot(obj).state
```

**Comment by Bonnevie** [comment]
Ah, convenient! Thanks for the tips. Will using `get_plot` also apply the `%%opts` of the cell or should options be applied using the class methods instead?
Apparently there is a package I did not know of that allows one to include svg in LaTeX, but I am guessing a broader range of output formats might still be helpful to me and others.

**Comment by philippjfr** [comment]
>Ah, convenient! Thanks for the tips. Will using get_plot also apply the %%opts of the cell or should options be applied using the class methods instead?

I won't, but since ``%%opts`` persists you could display the plot with the magic applied in one cell and then save the plot in a subsequent cell.
